config/201-clusterrole.yaml

```diff
@@ -35,6 +35,24 @@ rules:
   - patch
   - delete
 
+# Sources finalizer
+- apiGroups:
+  - sources.eventing.knative.dev
+  resources:
+  - awssqssources/finalizers
+  - containersources/finalizers
+  - cronjobsources/finalizers
+  - githubsources/finalizers
+  - kuberneteseventsources/finalizers
+  verbs: &everything
```

Comment:
change this to `verbs: *everything`

you are redefining the magic `everything` yaml tag